### PR TITLE
adding comments, and fixing edit for user and points

### PR DESCRIPTION
**Pull request by armenaa652**

### CODE REVIEW

File: app/controllers/points_controller.rb [hunk 1]
```diff
@@ -68,7 +68,7 @@ def update_index
   def update
     @point = Point.find(params[:id])
     if @point.update(point_params)
-      redirect_to
+      redirect_to(point_path(:id))
```
Comment (JonathanKonderla) [inline app/controllers/points_controller.rb:71]: nice catch

**Review by tonyxqing — APPROVED** [review]
Thanks for feeding the models, Adam. They were looking really skinny but you saved them from malnourishment. Not only that you thinned out our controllers which really helps balance the size of our codebase. Keep up the good work!

**Comment by dakrich** [comment]
Adam, you did a good job with this one I looked everything over and it all looks good. Keep it up we all believe you can keep typing more code!!!!!!!!

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -54,7 +54,7 @@ def update
     @current_user = User.find_by(id: session[:user_id])
 
     @user = User.find(params[:id])
-    if @user.update(params.require(:user).permit(:email, :isOfficer))
+    if @user.update(user_params)
```
Comment (JonathanKonderla) [inline app/controllers/users_controller.rb:57]: security issue

**Review by JonathanKonderla — CHANGES_REQUESTED** [review]

**Pull request merged**